### PR TITLE
Implement default filters in main menu for clients (#379), orders (#3…

### PR DESCRIPTION
**Pull request by benoitg**
### Changes proposed in this pull request:
- Implement default filters in main menu for clients (#379), orders (#378), and staff notes (#389)
### Status
- [ ] READY
- [ ] HOLD
- [X] WIP (Work-In-Progress)
### Deployment notes and migration
- [ ] Migration needed

@savoirfairelinux/mll

### CODE REVIEW

**Comment by manumilou** [comment]
A better way would be to use the `get_default_ops_value` function in the view.

For an example., have a look there: https://github.com/savoirfairelinux/sous-chef/blob/dev/src/member/views.py#L856
The filter value is default to TOBEPROCESSED value, if no get params are already set.

**Comment by benoitg** [comment]
@manumilou While that would also fulfil the requirements as written, I respectfully disagree it would be a better way.  Relying on get_default_ops_value would preclude having two menu items target two different filter configuration on the same model in the future (or at least force using two different mechanism to do so).  On a more general principle, I believe having restrictive filters in default configuration should be used sparingly (for example excuding tombstoned or expired data and things of that nature), if only because it can affect a lot of unrelated tests.

Normally I'd expect the menu view to interact with the relevant model views to inject the requested configuration.  Since we don't have a menu view, the best I found following existing pattern in the menu code, it to add it to the context manager.

Anyway, do feel free to change it if you really believe it's better, I won't be insulted.

File: src/sous-chef/templates/system/menu.html [hunk 1]
```diff
@@ -30,10 +30,10 @@ <h4 class="ui header name">
             </div>
         </form>
     </div>
-    <a class="item" href="{% url 'order:list' %}">{% trans 'Orders' %}
+    <a class="item" href="{% url 'order:list' %}?status={{ ORDER_FILTER_DEFAULT_STATUS | urlencode }}&delivery_date={{ ORDER_FILTER_DEFAULT_DATE | date:'SHORT_DATE_FORMAT' | urlencode }}">{% trans 'Orders' %}
```
Comment (manumilou) [inline src/sous-chef/templates/system/menu.html]: The date filter does not work with the formatting you provide.

`ORDER_FILTER_DEFAULT_DATE | date:'Y-m-d' `

**Comment by manumilou** [comment]
@benoitg Sorry for the delay ... I think you are actually right. The menu item should provide the query parameters. I just added a comment to fix the date format passed as parameter.

**Pull request merged**